CHANGELOG

```diff
@@ -4,9 +4,48 @@ qtile X.X.X, released XXXX-XX-XX:
         - added keybinding cheet sheet image generator
         - custom keyboardlayout display 
 
+qtile 0.15.0, released 2020-04-12
+    * bugfixes
+	- fix qtile reload (it was crashing)
+
 qtile 0.15.0, released 2020-04-12:
     !!! Config breakage !!!
         - removed the mpd widget, which depended on python-mpd.
+	- the Clock widget now requires pytz to handle timezones that are
+	  passed as string
+	- libqtile.command.Client does not exist anymore and has been
+	  replaced by libqtile.command_client.CommandClient
+    !!! deprecation warning !!!
+	- libqtile.command.lazy is deprecated in favor of libqtile.lazy.lazy
+    * features
+	- Python 3.8 support
+	- `wallpaper` and `wallpaper_mode` for screens
+	- bars can now have margins
+	- `lazy.toscreen` called twice will now toggle the groups
+	  (optional with the `toggle` parameter)
+	- `lazy.window.togroup` now has `switch_group` parameter to follow
+	  the window to the group it is sent to
+	- qtile now copies the default config if the config file does not exist
+	- all widgets now use Pango markup by default
+	- add an `fmt` option for all textbox widgets
+	- new PulseVolume widget for controlling PulseAudio
+	- new QuickExit widget, mainly for the default config
+	- new non-graph CPU widget
+	- KeyboardLayout widget: new `options` parameter
+	- CheckUpdates widget: support ArchLinux yay
+	- GroupBox widget: new `block_highlight_text_color` parameter
+	- Mpd2 widget: new `color_progress` parameter
+	- Maildir widget can now display the inbox grand total
+	- the Net widget can now use bits as unit
+	- Spacer widget: new `background_color` parameter
+	- More consistent resize behavior in Columns layout
+	- various improvements of the default config
+	- large documentation update and improvements (widget
```

Comment:
layouts screenshots still not integrated in documentation, just tool to make layout screenshots

Reply:
Damn!